homeassistant/components/adguard/sensor.py

```diff
@@ -89,17 +90,17 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up AdGuard Home sensor based on a config entry."""
-    adguard = hass.data[DOMAIN][entry.entry_id][DATA_ADGUARD_CLIENT]
+    data: AdGuardData = hass.data[DOMAIN][entry.entry_id]
 
     try:
-        version = await adguard.version()
+        version = await data.client.version()
     except AdGuardHomeConnectionError as exception:
         raise PlatformNotReady from exception
```

Comment:
We also do this call in the init, maybe we can deduplicate this call, since this call is not platform specific